<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageEditorDialog</class>
 <widget class="QDialog" name="ImageEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Overwrite Image...</string>
  </property>
  <layout class="QVBoxLayout" name="vlInterface">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="labPicture">
     <property name="minimumSize">
      <size>
       <width>516</width>
       <height>288</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="buttomFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="vlButtons">
      <item>
       <layout class="QHBoxLayout" name="hlButtons">
        <item>
         <widget class="QPushButton" name="cmdReplace">
          <property name="toolTip">
           <string>Import picture</string>
          </property>
          <property name="text">
           <string>&amp;Import...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="hsButtons">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="cmdSave">
          <property name="toolTip">
           <string>Apply changes</string>
          </property>
          <property name="text">
           <string>&amp;Overwrite</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cmdClose">
          <property name="toolTip">
           <string>Discard changes</string>
          </property>
          <property name="text">
           <string>&amp;Close</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
